src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -154,6 +157,33 @@ class DocBaseViewer extends BaseViewer {
         super.destroy();
     }
 
+    /**
+     * Converts a value and unit to page number
+     *
+     * @return {number|undefined} a page number > 0
+     */
+    getStartPage() {
+        let convertedValue;
+
+        const { unit, value } = this.startAt;
+
+        if (!value || !unit) {
+            return convertedValue;
+        }
+
+        if (unit === PAGES_UNIT_NAME) {
+            convertedValue = parseInt(value, 10);
+            if (!convertedValue || convertedValue < 1) {
+                // Negative values aren't allowed, fall back to default behavior
+                return undefined;
+            }
+        } else {
+            console.error('Invalid unit for start:', unit); // eslint-disable-line no-console
```

Comment:
Should we be surfacing this error in some other way other than as a console.error?